samples/ccr/start.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+set -e
+
+# Start metrics store
+docker-compose -f ./docker-compose-metricstore.yml up -d --remove-orphans
+
+# Start Elasticsearch
```

Comment:
There's a few consistent typos in this file. I think you'll want to make these changes:
1. Elasticsearch/ES &#8594; OpenSearch 
1. Rally &#8594; OpenSearch Benchmark
1. /home/ankikala &#8594; ${HOME}

Reply:
Done.